app/controllers/users_controller.rb

```diff
@@ -42,9 +39,9 @@ def new
   def create
     @user = User.new user_params
     if @user.save
-      log_in @user
-      flash[:success] = t("create_user")
-      redirect_to @user
+      @user.send_activation_email
```

Comment:
cho anh biết rails gửi mail như thế nào nhé

Reply:
rails gui mail thong qua SMTP do do can cau hinh SMTP trong file environments/development.rb
config.action_mailer.delivery_method = :smtp
# SMTP settings for gmail
config.action_mailer.smtp_settings = {
 :address              => "smtp.gmail.com",
 :port                 => 587,
 :user_name            => ENV['gmail_username'],
 :password             => ENV['gmail_password'],
 :authentication       => "plain",
:enable_starttls_auto => true
}

luu cac thong tin nhay cam su dung gem figaro

+  def account_activation user
+    @user = user
+    mail to: user.email, subject: "Account activation"
phuong thuc tren se lay parameter  cua user va gui mail su dung mail de gui toi dia chi mail cua user